server/src/app.js

```diff
@@ -0,0 +1,29 @@
+import express from 'express'
+import path from 'path'
+import session from 'express-session'
+import connectPgSimple from 'connect-pg-simple'
+import api from './routes/api'
+import passport from './passport'
+import config from './config'
+
+const PgSession = connectPgSimple(session)
+const app = express()
+
+app.use(session({
+  secret: 'secret',
```

Comment:
This secret should be configurable as well?

Reply:
Oh yes! I already fixed this but i will leave it for the next PR.